src/CSharpMongoMigrations/CSharpMongoMigrations.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFrameworks>netstandard2.1;net8.0</TargetFrameworks>
```

Comment:
Hi. Thank you for your PR. However, could you please rollback this line. 
Changing the target Framework version is leading to changes in the Nuget package. 
I'm not going to change package distribution yet.